src/components/Setup/index.tsx

```diff
@@ -30,7 +31,8 @@ const Setup: React.FC = () => {
   const intl = useIntl();
   const [isUpdating, setIsUpdating] = useState(false);
   const [currentStep, setCurrentStep] = useState(1);
-  const [plexSettingsComplete, setPlexSettingsComplete] = useState(false);
+  const [msSettingsComplete, setMSSettingsComplete] = useState(false);
```

Comment:
```suggestion
  const [mediaServerSettingsComplete, setMediaServerSettingsComplete] = useState(false);
```
I think that the use of the abberviations `ms`/`MS` isn't necessary, and that clearer naming is better since we have other things with `mediaServer`/`MediaServer` in their naming.